Monaray97 patch 1

monaray97 opened the pull request:
made some edits to add more info about the M365 app & activities in the conceptual connector docs since both are GA
added a placement doc for the steps to deploy a connector in the Teams Admin center. code for this will be live by EO june.

Lauragra commented on concepts/connecting-external-content-deploy-teams-admin-center:
```diff
@@ -0,0 +1,11 @@
+ms.prod: search
+---
+
+# Deploy your connector in the Teams Admin Center
```
I'm not clear about what the intention is for this topic since it doesn't have any content. Can you help to clarify?

monaray97 replied:
we have a draft of the documentation we plan to review with a content writer tomorrow. since i was making this PR to address other changes, i figured i could also add a new page as a placeholder right now. this page introduces a new feature/capability for ISVs to easily deploy their connector in the teams admin center. the feature will go out to public preview in july.

Lauragra replied:
Okay, I'll wait to review and approve until you've added the content here.

Lauragra commented on concepts/connecting-external-content-manage-connections.md:
```diff
@@ -74,7 +74,7 @@ To enable one or more content experiences, set the value of the **enabledContent
 | enabledContentExperiences value | Description |
 |-|-|
-| search | Allows your content to appear in Microsoft search results. The format of these results is consistent across different search canvases, such as SharePoint and Microsoft Bing. |
+| search | Allows your content to appear in Microsoft search results. The format of these results is consistent across different search canvases, such as SharePoint and Microsoft Bing. Your content will also appear in the Quick Access & My Content sections of the [M365 App](Office.com) if you send [activities](./graph/api/externalconnectors-externalitem-addactivities) and register [activity setti ...
```
Per branding guidelines, we always spell out Microsoft 365 (not M365).

monaray97 replied:
gotchu

monaray97 commented:
hi @Lauragra just made the requested changes. lmk if there is anything else you need from me

monaray97 commented:
Hi @Lauragra, I removed the doc that doesn't have any content. Instead, I added a new doc with new content and made some other updates to the other conceptual docs (removing anything related to compliance). Please review and let me know if any more changes are needed.

ajacks-msft commented on concepts/connecting-external-content-experiences:
```diff
@@ -0,0 +1,55 @@
+
+# Microsoft Graph connectors experiences
+
+Microsoft Graph connectors offer an intuitive way to bring content from external services into Microsoft Graph and the Microsoft Search index, which enables external data to be discoverable across a set of Microsoft 365 experiences.
```
I suggest using the Semantic Index brand.

ajacks-msft commented on concepts/connecting-external-content-experiences:
```diff
@@ -0,0 +1,55 @@
+## Microsoft Search 
+[Microsoft Search](microsoftsearch/connectors-overview) with Graph connectors allows your third-party data to be searchable for users in the Microsoft Search endpoints which include Office.com, Bing at Work, and SharePoint. 
+
+To congifure your connector for Microsoft Search, you must:
```
spelling of "configure"

ajacks-msft commented on concepts/connecting-external-content-experiences:
```diff
@@ -0,0 +1,55 @@
+>Also, only customers who are part of the [Copilot Early Access Program](https://blogs.microsoft.com/blog/2023/05/09/introducing-the-microsoft-365-copilot-early-access-program-and-2023-microsoft-work-trend-index/) will have Copilot enabled in their tenants in the coming months.
+
+To configure your connector for Copilot, you must:
+- Add a [urlToItemrResolver](graph/api/resources/externalconnectors-urltoitemresolverbase) in [activitySettings](graph/api/resources/externalconnectors-activitysettings) when you [create your connection](graph/connecting-external-content-manage-connections#create-a-connection). A `urlToItemResolver` will enable the platform to become aware when users share URLs from your external content with eac ...
```
Possible spelling of "urlToItemResolver"

ajacks-msft commented on concepts/connecting-external-content-experiences:
```diff
@@ -0,0 +1,55 @@
+>Microsoft Graph connectors with Microsoft 365 Copilot is in early access preview. When you get access to the [developer preview program](https://forms.office.com/Pages/ResponsePage.aspx?id=v4j5cvGGr0GRqy180BHbR06na7ADVcFBoFXEJ5M8YD1UNkdQODE5VDNKOUQ2UlBSVUJJRlJDVktTVS4u), you'll receive instructions on how to test your Graph connector with Microsoft 365 Copilot.
+>Also, only customers who are part of the [Copilot Early Access Program](https://blogs.microsoft.com/blog/2023/05/09/introducing-the-microsoft-365-copilot-early-access-program-and-2023-microsoft-work-trend-index/) will have Copilot enabled in their tenants in the coming months.
+
+To configure your connector for Copilot, you must:
```
If all of these steps are technically optional, the "must" language should be changed. "To make sure Copilot uses your content effectively, you can:"

ajacks-msft commented on concepts/connecting-external-content-experiences:
```diff
@@ -0,0 +1,55 @@
+- Add [user activities](/graph/api/externalconnectors-externalitem-addactivities) on your items. Supported activity types are shown [here](/graph/api/resources/externalconnectors-externalactivity). These activities inform the platform of which users have interacted most recently with which items. Items with activities performed by a given user will have a higher likelihood of being displayed to t ...
+- Add a [urlToItemrResolver](graph/api/resources/externalconnectors-urltoitemresolverbase) in [activitySettings](graph/api/resources/externalconnectors-activitysettings) when you [create your connection](graph/connecting-external-content-manage-connections#create-a-connection). A `urlToItemResolver` will enable the platform to become aware when users share URLs from your external content with eac ...
+
+## Copilot (coming soon)
```
Use "Microsoft 365 Copilot" at least the first time you mention it.

Lauragra commented on concepts/connecting-external-content-connectors-overview.md:
```diff
@@ -9,19 +9,20 @@ ms.prod: search
+Today, with Microsoft Graph connectors, the data you bring in from your organization can be discoverable in the following [experiences](connecting-external-content-experiences):
+- [Microsoft Search](microsoftsearch/overview-microsoft-search) (GA)
+- [Context IQ in Outlook Web](https://techcommunity.microsoft.com/t5/microsoft-365-blog/microsoft-editor-using-context-iq-in-outlook-on-the-web-and-word/ba-p/3643497) (GA)
+- [Copilot](https://blogs.microsoft.com/blog/2023/03/16/introducing-microsoft-365-copilot-your-copilot-for-work/) (preview)
```
Do we need to refer to this as "Microsoft 365 Copilot"?

monaray97 replied:
yes, we should

Lauragra commented on concepts/connecting-external-content-connectors-overview.md:
```diff
@@ -9,19 +9,20 @@ ms.prod: search
+Today, with Microsoft Graph connectors, the data you bring in from your organization can be discoverable in the following [experiences](connecting-external-content-experiences):
+- [Microsoft Search](microsoftsearch/overview-microsoft-search) (GA)
+- [Context IQ in Outlook Web](https://techcommunity.microsoft.com/t5/microsoft-365-blog/microsoft-editor-using-context-iq-in-outlook-on-the-web-and-word/ba-p/3643497) (GA)
+- [Copilot](https://blogs.microsoft.com/blog/2023/03/16/introducing-microsoft-365-copilot-your-copilot-for-work/) (preview)
```
I added preview here since I think you said that there's a public preview? What is the ETA?

monaray97 replied:
end of July

Lauragra commented on concepts/connecting-external-content-connectors-overview.md:
```diff
@@ -9,19 +9,20 @@ ms.prod: search
+- [Microsoft Search](microsoftsearch/overview-microsoft-search) (GA)
+- [Context IQ in Outlook Web](https://techcommunity.microsoft.com/t5/microsoft-365-blog/microsoft-editor-using-context-iq-in-outlook-on-the-web-and-word/ba-p/3643497) (GA)
+- [Copilot](https://blogs.microsoft.com/blog/2023/03/16/introducing-microsoft-365-copilot-your-copilot-for-work/) (preview)
+- [Microsoft 365 Hub](https://office.com) (formerly known as [Office.com](https://office.com)) (coming soon)
```
We don't generally mention or make a promise of future functionality in the docs. Do we have an ETA for this?

monaray97 replied:
end of July

Lauragra replied:
Okay, please be sure to update the content at the end of July to reflect that this is available.

Lauragra commented on api-reference/beta/resources/externalconnectors-externalconnection.md:
```diff
@@ -34,11 +34,10 @@ A logical container to add content from an external source into Microsoft Graph.
 | connectorId               | String            | The Teams App ID. Optional.|
 | description               | String           | Description of the connection displayed in the Microsoft 365 admin center. Optional. |
-| enabledContentExperiences | microsoft.graph.externalConnectors.contentExperienceType collection | The list of content experiences the connection will participate in. Possible values are `search` and `compliance`.|
+| enabledContentExperiences | microsoft.graph.externalConnectors.contentExperienceType collection | The list of content experiences the connection will participate in. Possible values are `search`.|
```
This is an API-level change that warrants a mention in the changelog. You need to create a ChangeList record that mentions that the compliance resource was removed, and that this value was added. For details, see: https://dev.azure.com/msazure/One/_wiki/wikis/Microsoft%20Graph%20Partners/211742/Changelog

monaray97 replied:
done

Lauragra commented on concepts/connecting-external-content-connectors-overview.md:
```diff
@@ -9,19 +9,20 @@ ms.prod: search
 
-Today, with Microsoft Graph connectors, the data you bring in from your organization can appear in Microsoft Search results. This feature expands the types of content sources that are searchable in your Microsoft 365 productivity apps and the broader Microsoft ecosystem. Soon, connectors will power many other Microsoft 365 intelligent experiences, such as Viva Topics.
+Today, with Microsoft Graph connectors, the data you bring in from your organization can be discoverable in the following [experiences](/graph/connecting-external-content-experiences):
+- [Microsoft Search](microsoftsearch/overview-microsoft-search) (GA)
```
Let's delete the (GA) here and in the next line because we can assume that if it doesn't say preview or coming soon, it's generally available.

monaray97 replied:
done

BrianTJackett commented:
@monaray97 based on recent discussions with marketing, recommend update "Microsoft 365 Hub" -> "Microsoft 365 App".  If need to do that in separate PR after this that is ok as well.
